api/Console.json

```diff
@@ -0,0 +1,1048 @@
+                "Alias (console.exception) not available"
+              ]
+            },
+            "safari": {
+              "version_added": true,
+              "notes": [
+                "Alias (console.exception) not available"
+              ]
+            },
+            "safari_ios": {
+              "version_added": null,
+              "notes": [
+                "Alias (console.exception) not available"
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "group": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/group",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "4",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "notes": [
+                "Available in workers from 38 onwards"
+              ]
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "4"
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "groupCollapsed": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/groupCollapsed",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "6"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "9",
+              "notes": [
+                "This method is supported starting in Gecko 9 (Firefox 9 / Thunderbird 9 / SeaMonkey 2.6), but the new group was not collapsed; it's functionally was equivalent to console.group(). Since Gecko 52.0 (Firefox 52.0 / Thunderbird 52.0 / SeaMonkey 2.49) the group is collapsed",
```

Comment:
So basically it works starting with 52. Lets remove this note and add version_added: 52.